packages/project/src/android/project.ts

```diff
@@ -49,11 +48,60 @@ export class AndroidProject {
     return null;
   }
 
+  /**
+   * Update the Android package name. This renames the package in `AndroidManifest.xml`,
+   * the `applicationId` in `app/build.gradle`, and renames the java
+   * package for the `MainActivity.java` file.
+   * 
+   * This action will mutate the project on disk!
+   */
   async setPackageName(packageName: string) {
+    const oldPackageName = await this.manifest.getDocumentElement()?.getAttribute('package');
     this.manifest.getDocumentElement()?.setAttribute('package', packageName);
     await this.appBuildGradle?.setApplicationId(packageName);
+
```

Comment:
I think we should also modify manifest's activity attribute (android:name). This should do:

```
this.manifest.setAttrs('manifest/application/activity', {
    'android:name': `${packageName}.MainActivity`,
});
```